test/Microsoft.AspNetCore.Razor.Evolution.Test/TagHelperBinderSyntaxTreePassTest.cs

```diff
@@ -55,8 +55,16 @@ public void Execute_RewritesTagHelpers()
             Assert.Equal("input", inputTagHelper.TagName);
         }
 
-        [Fact]
-        public void Execute_RewritesTagHelpers_TagHelperMatchesElementTwice()
+        [Theory]
```

Comment:
No theories, just re-write the test. Can always store the boiler plate descriptors in a property.